libraries/Microsoft.Bot.Builder.Prompts/Microsoft.Bot.Builder.Prompts.csproj

```diff
@@ -4,6 +4,13 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Recognizers.Text.DateTime" Version="1.0.0.36" />
+    <PackageReference Include="Microsoft.Recognizers.Text.Number" Version="1.0.0.36" />
+    <PackageReference Include="Microsoft.Recognizers.Text.NumberWithUnit" Version="1.0.0.36" />
+    <PackageReference Include="Microsoft.Recognizers.Text.Sequence" Version="1.0.0.36" />
+  </ItemGroup>
+
```

Comment:
Did ya notice that these packages are (as of Monday) .Net Standard 2?? The Build warnings are gone!